### DISCUSSION TITLE
Can the routr be used as a video conferencing server?

### DISCUSSION THREAD

**Discussion by trojan-tj** [fonoster/yaps#443] [General]
Hello! I have a need to organize a server for video communication of several people using the SIP protocol (including couple of polycom hardware video codecs). Is it possible to organize this through a router server? Or with the help of a routr, only audiotelephony is possible?

**Comment by psanders** [comment]
@trojan-tj Routr along will cannot be use for video conferencing.

Routr does not deal with media handling (only with the signaling.)

You will need something like Jitsi Meet or Janus.

I have personally used Jitsi Meet and it's really good software.

If could also combine Routr + Jitsi Meet using Jigasi if you still need SIP.
